Dockerfile

```diff
@@ -203,6 +203,20 @@ RUN curl -Lo kind \
    && chmod +x kind \
    && mv kind /usr/local/bin/kind
 
+ARG RAT_VERSION="0.13"
+
+ENV RAT_VERSION="${RAT_VERSION}" \
+    RAT_JAR="/tmp/apache-rat-${RAT_VERSION}.jar" \
```

Comment:
Can we put it in another directory? For example `/opt/`. The /tmp/ directory is by definition a ephemeral directory, but we want this file to exist longer.

Reply:
Sure. I will correct it for hive/minicluster and others above.